src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content=" width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Product cards</title>
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Imac image"
+        class="card__image"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating" >
+        <div
+        class="card__stars stars stars--4"
+        >
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-value">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >
+        <p class="card__button-text">buy</p>
+      </a>
```

Comment:
I think there's no need to add a separate <p> element for the text - you can just add the word 'buy' to anchor element.

Reply:
Forgot to delete))) just wanted to style the text separately